launch_testing/launch_testing/__init__.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         self.name = name
         self.launch_descriptor = launch_descriptor
         self.expected_lines = expected_lines
+        self.expected_output = b'\n'.join(self.expected_lines)
```

Comment:
The use case is in the pendulum demo, in which I expect to see a particular block of output printed at the end of the demo, e.g. 

```
rttest statistics:
  - Minor pagefaults: 48
  - Major pagefaults: 0
  Latency (time after deadline was missed):
    - Min: 4218 ns
    - Max: 23552 ns
    - Mean: 5735.88 ns
    - Standard deviation: 614.286


PendulumMotor received 261 messages
PendulumController received 262 messages
```

Accepting multiple single-line regexes is not adequate in this situation, since if the program exits early before getting to the last line due to an error, the test will still pass.

We should eventually work towards implementing multiple single-line regexes as well and selecting between modes. I do not currently have an need for it, however, and the functionality proposed is a superset of the existing functionality.